routes/index.js

```diff
@@ -213,4 +213,84 @@ router.post('/save-journal', async function(req, res, next) {
 
 });
 
+/*
+
+put request expects req.body with following format:
+
+ {"section": 
+        {
+            
+            "journal_reference_id": XX, 
+            "section_number": XX,
+            "contentDetails": {
+                "content": "CONTENT" 
+            }
+        }
+      }
+*/
+
+router.post('/save-section', async function(req, res, next){
+  //initiates connection to database
+  const client = await pool.connect()
+
+  //parses data from request body
+  const referenceId = req.body.section.journal_reference_id;
+  const content = req.body.section.contentDetails.content;
+  const sectionNumber = req.body.section.section_number;
+
+  //database query for journal_sections
+  const databaseSectionsQuery = 'INSERT INTO journal_sections (journal_reference_id, section_number) VALUES ($1, $2) RETURNING *'
+  
+  const sectionValues = [referenceId, sectionNumber];
+
+  //database query for journal_content
+  const databaseContentQuery = 'INSERT INTO journal_content (journal_section_id, content) VALUES ($1, $2) RETURNING *'
+
+  let responseObject = {sections: {}}
```

Comment:
There isn't really a need for the response to include the `sections` property, since we only really care about whether the action we wanted to perform was a success or something else (e.g. an error) in the context of the frontend